build.sbt

```diff
@@ -105,18 +105,23 @@ lazy val validation = (project in file("validation"))
 
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
+  .settings(Revolver.settings:_*)
   .settings(
-    assemblyMergeStrategy in assembly := {
-      case "application.conf" => MergeStrategy.concat
-      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
-      case x =>
-        val oldStrategy = (assemblyMergeStrategy in assembly).value
-        oldStrategy(x)
-    },
-    libraryDependencies ++= akkaPersistenceDeps ++ httpDeps
+    Seq(
+      assemblyJarName in assembly := {s"${name.value}.jar"},
```

Comment:
You're right, though. This is what worked for me:
```
$ sbt
> project panel
> assembly
```
... which saves the .jar to `panel/target/scala-2.12/panel.jar`.

Then, with the API (and Cassandra and Zookeeper) running, this command runs the loader successfully:

```
java -jar  panel/target/scala-2.12/panel.jar [path-to-project]/panel/src/main/resources/inst_data_2017_dummy.csv
```